benchmark/vm2_fiber_chain.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+
+require 'benchmark'
```

Comment:
Any scripts in `benchmark/*.rb` or `benchmark/*.yml` are [expected to be runnable as benchmark-driver's input](https://github.com/ruby/ruby/blob/df58e74e0254c329e8c4fe748a266910070a71a1/common.mk#L1118). Could you change this script for benchmark-driver's format (at least that wouldn't `require 'benchmark'`) or create sub-directory for this under `benchmark/`?

Reply:
Thanks, I will fix it.

Reply:
So, I'm happy to change this.

However, right now I'm using `time = Benchmark.realtime do ...` to measure the time of specific parts of the script. I can remove this, or is there a better alternative?